erpnext/stock/doctype/material_request/test_material_request.py

```
@@ -1118,4 +1118,65 @@ def make_material_request(**args):
 	return mr
 
 
+
 EXTRA_TEST_RECORD_DEPENDENCIES = ["Currency Exchange", "BOM"]
+
+test_dependencies = ["Currency Exchange", "BOM"]
+test_records = frappe.get_test_records("Material Request")
+
+
+class TestMaterialRequest(FrappeTestCase):
```

Ruff (0.14.10)

Redefinition of unused `TestMaterialRequest` from line 26

(F811)